jkind/src/jkind/JKindSettings.java

```diff
@@ -15,13 +16,20 @@
 	public int pdrMax = 1;
 	public boolean inductiveCounterexamples = false;
 	public boolean reduceIvc = false;
+	public boolean allIvcs = false;
 	public boolean smoothCounterexamples = false;
+	public int allIvcsAlgorithm = 1;
+	public int allIvcsMaxGrows = 1000;
+	public int allIvcsJkindTimeout = -1; // if set to -1, a timeout will be computed based on the first call of the jkind's solve method
 	public boolean inlining = true;
 	public boolean slicing = true;
-
 	public SolverOption solver = SolverOption.SMTINTERPOL;
 	public boolean scratch = false;
-
 	public String writeAdvice = null;
 	public String readAdvice = null;
+	public boolean allAssigned = false;
+	public String useUnsatCore = null;
```

Comment:
```useUnsatCore``` is a string value as it can be set to the path to an input file containing existing ```IVCs``` to start the over approximation with IVC (line 60 of /jkind/src/jkind/JSupport.java). It will be set to not null only for testing purpose as the default is null.

Reply:
Removed ```useUnsatCore``` as it's no longer used after the removal of JKind.java:
https://github.com/janetlj/jkind/commit/0e487c7e7119c875c1ae922948dca109077b3b31